pages/1.11/installing/oss/patch/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: layout.pug
+navigationTitle: Patching
+title: Patching
+menuWeight: 3
+excerpt: 
+---
+
+# Patching live clusters with No downtime
+
+A DC/OS Patch describes a set of changes and supporting data designed to update, fix, or improve the features/functionality of DC/OS. A point release that consists of minor changes is also called a Patch. 
```

Comment:
Change "A DC/OS Patch" to "A DC/OS patch"

Reply:
Done.